### DISCUSSION TITLE
Underlined title/anchor links in markdown rendered pages

### DISCUSSION THREAD

**Discussion by nuxy** [community/community#68483] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

As stated in the title, the update occurred within the last couple days.  Seems to affect all markdown rendered pages.

Does anyone know if this is a intended "feature", or just a bug? I cannot find any information related to the change.

**Comment by chk1** [comment]
I noticed this as well today

**Comment by nuxy** [comment]
Ok - I wasn't crazy.  It was a bug and _it's now fixed_.

Closing this thread.

**Reply by chk1** [reply, marked as answer]
I got this notification today 
![image](https://github.com/community/community/assets/828496/3390e3c8-bc0e-49e2-823a-2b2ce8273c7d)

>Link underlines
>By default, links in text blocks on GitHub will now be underlined. You can set your preference in the accessibility settings (https://github.com/settings/accessibility).

The accessibility setting "Link underlines" is marked with "Beta" though, so it was deliberate after all

**Reply by nuxy** [reply]
Makes sense for _paragraph text blocks_ and related elements.  However, the **underlined `# Title Text`** didn't make much sense.

Good thing they are resorting to the _forme_r vs **latter**.  The UI inconsistencies was killing my :eyes:
